client/lib/mixins/i18n/index.js

```diff
@@ -259,6 +259,10 @@ function getTranslationFromJed( options ) {
 	var jedMethod = 'gettext',
 		jedArgs;
 
+	if ( ! i18nState.jed ) {
```

Comment:
I have no experience with this lib so I also have no idea if that change is safe :)

Would it make sense to refactor test to use exported plain component and simply pass `identity` as `translate` prop? I'm not convinced, just trying to make mocking experience easier. I didn't think of downsides too much.